cmd/sovereignnode/sovereignNodeRunner.go

```diff
@@ -1048,7 +1049,7 @@ func createOutGoingTxDataSigners(signingHandler consensus.SigningHandler) (bls.E
 			return nil, err
 		}
 
-		endRoundExtraSignerOutGoingTx, err := extraSigners.NewSovereignSubRoundEndExtraSigner(extraSignerHandler, mbType)
+		endRoundExtraSignerOutGoingTx, err := extraSigners.NewSovereignSubRoundEndExtraSigner(extraSignerHandler, mbType, enableEpochsHandler)
```

Comment:
everything needs to be activated by design.

Reply:
Yes, agree, it will be done this way, as mentioned in previous PRs, this is for viebechain upgrade